src/main/java/frc/robot/subsystems/MunecaSubsystem.java

```diff
@@ -1,51 +1,51 @@
 package frc.robot.subsystems;
 
+import com.revrobotics.spark.SparkBase;
 import com.revrobotics.spark.SparkBase.PersistMode;
 import com.revrobotics.spark.SparkBase.ResetMode;
+import com.revrobotics.spark.SparkClosedLoopController;
 import com.revrobotics.spark.SparkLowLevel.MotorType;
 import com.revrobotics.RelativeEncoder;
 import com.revrobotics.spark.SparkMax;
 import com.revrobotics.spark.config.SparkBaseConfig;
 import com.revrobotics.spark.config.SparkMaxConfig;
 
+import edu.wpi.first.math.MathUtil;
 import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 import edu.wpi.first.wpilibj2.command.InstantCommand;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 import frc.robot.Constants.DriveConstants;
 import frc.robot.Constants.MunecaConstants;
 import frc.robot.Constants.NEOMotorsConstants;
 
-public class MunecaSubsystem extends SubsystemBase {
-  private final SparkMax motor;
+public class MunecaSubsystem extends SubsystemBase implements MovableSubsystem {
+  private final SparkMax motor = new SparkMax(MunecaConstants.MUNECA_MOTOR_ID, MotorType.kBrushless);
+  private final SparkClosedLoopController controller = motor.getClosedLoopController();
   private final RelativeEncoder munecaEncoder;
-
-  private static final double SLOWDOWN_RANGE = 7.0;
-  private static final double UPPER_LIMIT = 0.0;
-  private static final double LOWER_LIMIT = -15.0;
-  public static final String DASH_MUNECA_POS = "Muñeca Posicion";
+  private static final double UPPER_LIMIT = 0;
+  private static final double LOWER_LIMIT = -20;
+  public static final String DASH_MUNECA_POS = "Muneca Posicion";
   public static final String DASH_RESET_MUNECA_ENCODER = "Reiniciar Encoder Muñeca";
+  private double position;
 
   public MunecaSubsystem() {
-    motor = new SparkMax(MunecaConstants.MUNECA_MOTOR_ID, MotorType.kBrushless);
-
     motor.setCANTimeout(DriveConstants.CAN_TIMEOUT);
-
     motor.configure(getLeaderConfig(), ResetMode.kResetSafeParameters, PersistMode.kPersistParameters);
-
     munecaEncoder = motor.getEncoder();
-    munecaEncoder.setPosition(0);
-
-    SmartDashboard.putData(DASH_RESET_MUNECA_ENCODER, new InstantCommand(() -> munecaEncoder.setPosition(0)));
+    reset();
+    SmartDashboard.putData(DASH_RESET_MUNECA_ENCODER, new InstantCommand(this::reset));
   }
 
   private static SparkMaxConfig getLeaderConfig() {
     SparkMaxConfig leaderConfig = new SparkMaxConfig();
 
-    leaderConfig.softLimit
-      .forwardSoftLimitEnabled(true)
-      .forwardSoftLimit(0)
-      .reverseSoftLimitEnabled(true)
-      .reverseSoftLimit(-15);
+    leaderConfig.closedLoop
+      .p(0.01)
+      .i(0)
+      .d(0)
+      .maxMotion
+      .maxVelocity(0.5)
+      .maxAcceleration(0.1);
```

Comment:
```suggestion
    leaderConfig.closedLoop
      .p(0.02)
      .i(0)
      .d(0)
      .maxMotion
      .maxVelocity(0.4)
      .maxAcceleration(0.05);
```

Estuvimos probando hoy y encontramos que estos valores son mejores :)